website/views/options/settings.jade

```diff
@@ -397,7 +397,7 @@ script(id='partials/options.settings.subscription.html',type='text/ng-template')
         div(ng-if='!user.purchased.plan.customerId || (user.purchased.plan.customerId && user.purchased.plan.dateTerminated)')
           h4(ng-if='(user.purchased.plan.customerId && user.purchased.plan.dateTerminated)')= env.t("resubscribe")
           .form-group.reduce-top-margin
-            .radio(ng-repeat='block in Content.subscriptionBlocks | toArray | omit: "discount==true" | orderBy:"months"')
+            .radio(ng-repeat='block in Content.subscriptionBlocks | toArray | omit: "discount==true" | orderBy:"months"', ng-if="block.key !== 'group_monthly'")
```

Comment:
or use a new property on `block` to distinguish between users and groups plans like `group.type === 'user'`. Or even split the subscriptionBlocks obj in two different objects